make/tools.mk

```diff
@@ -565,15 +565,14 @@ endif
 
 # OPENSSL download URL
 ifdef WINDOWS
-  openssl_install: OPENSSL_URL  := http://slproweb.com/download/Win32OpenSSL-1_0_2c.exe
+  openssl_install: OPENSSL_URL  := https://slproweb.com/download/Win32OpenSSL-1_0_2d.exe
```

Comment:
This should go in another PR. Or at least in another commit. I want to push hard on making commits and PRs as atomic as possible.